ChromProcess/Loading/peak/peak_from_file.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+
+def peak_rt_from_file(chromatogram, Peakfile):
+    '''
+    Get peak retention time indices from a PeakCollections file.
+    chromatogram: Chromatogram object
+        Chromatogram object to which the peaks should be added
+    Peakfile: string
+        Location of peak collection csv
+    
+    Returns
+    -------
+    peak_indices: ndarray  
+        array of peak indices
+    '''
+    import pandas as pd
```

Comment:
It should be possible to replace the pandas loading with the peak_collection_from_csv() in the Loading module. Peak information can be accessed from the PeakCollection object.